src/chainparams.cpp

```diff
@@ -334,9 +335,15 @@ CChainParams* CChainParams::Factory(CBaseChainParams::Network network, CScript s
     }
 }
 
+CChainParams &Params(CBaseChainParams::Network network)
+{
+    globalSwitchingChainParams.reset(CChainParams::Factory(network, CScript()));
+    return *globalSwitchingChainParams;
+}
+
```

Comment:
Ah, but if your goal is easier rebases, I think that should take priority. Esp. since `Params()` will go away eventually.

Thoughts?